tool/src/kotlin/mod.rs

```diff
@@ -431,6 +447,11 @@ impl<'a, 'cx> TyGenContext<'a, 'cx> {
             Type::Callback(_) => {
                 format!("DiplomatCallback_{}_Native", additional_name.unwrap()).into()
             }
+            Type::ImplTrait(ref trt) => {
+                let trait_id = trt.id();
+                let resolved = self.tcx.resolve_trait(trait_id);
+                format!("DiplomatTrait_{}_Wrapper_Native", resolved.name).into()
```

Comment:
I've left them as-is because now these types are internal, and the user would only interact with the trait itself, which just has the original name from rust (`TesterTrait` in this example)

Reply:
yeah I'm still not a fan of breaking naming conventions

@jcrist1 if you would like to do some uniform renaming I think that would be welcome.